test/e2e/support/utils/click-on-more-menu-item.js

```diff
@@ -7,7 +12,19 @@ export async function clickOnMoreMenuItem( buttonLabel ) {
 	await expect( page ).toClick(
 		'.edit-post-more-menu [aria-label="Show more tools & options"]'
 	);
-	await page.click(
-		`.edit-post-more-menu__content button[aria-label="${ buttonLabel }"]`
-	);
+	const moreMenuContainerSelector =
+		'//*[contains(concat(" ", @class, " "), " edit-post-more-menu__content ")]';
+	let elementToClick = first( await page.$x(
+		`${ moreMenuContainerSelector }//button[contains(text(), "${ buttonLabel }")]`
+	) );
+	// If button is not found, the label should be on the info wrapper.
+	if ( ! elementToClick ) {
+		elementToClick = first( await page.$x(
+			moreMenuContainerSelector +
+			'//button' +
+			`/*[contains(concat(" ", @class, " "), " components-menu-item__info-wrapper ")][contains(text(), "${ buttonLabel }")]`
+		) );
+	}
+	expect( elementToClick ).not.toBeFalsy();
```

Comment:
Does this line is really required? It might fail on the next line regardless.

Reply:
The line was removed 👍